metaci/build/tests/test_models.py

```diff
@@ -48,10 +48,40 @@ def test_planrepo_find_on_build_init(self):
         build = Build(repo=repo, plan=plan)
         assert build.planrepo == planrepo
 
+    @mock.patch("metaci.repository.models.Repository.github_api")
+    @mock.patch("metaci.cumulusci.keychain.MetaCIProjectKeychain.get_org")
+    def test_run(self, get_org, gh):
+        cwd = os.getcwd()
+
+        # mock github zipball
+        def archive(format, zip_content, ref):
+            with open(Path(__file__).parent / "testproject.zip", "rb") as f:
+                zip_content.write(f.read())
+
+        gh.archive.side_effect = archive
+
+        # mock org config
+        org_config = OrgConfig({}, "test")
+        org_config.refresh_oauth_token = mock.Mock()
+        get_org.return_value = org_config
+
+        build = BuildFactory()
+        build.plan.flows = "test"
+
+        try:
+            build.run()
+        finally:
+            detach_logger(build)
+            os.chdir(cwd)
```

Comment:
If this is the only place the `cwd` variable is being referenced I think it would be a tiny bit better if the variable declaration was closer to this location.

Reply:
I actually want to switch to an autouse fixture that resets the cwd after each test -- like this but without the class scope: https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/conftest.py#L32

Reply:
@Br4nd0R done and ready to re-review